modules/pam_access/access.conf.5.xml

```diff
@@ -179,6 +179,11 @@
     </para>
     <para>-:root:ALL</para>
 
+    <para>
+      User with uid <emphasis>1003</emphasis> and group with gid <emphasis>1000</emphasis> should be allowed to get access
```

Comment:
Please split this overly long line. Also should it be rather 'An user' and 'a group'?